### ISSUE TITLE
selector-pseudo-element-case

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1058]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Valid:
```css
body::BEFORE {
    background-color: red;
    content: ' + ';
}
```

```css
body::before {
    background-color: red;
    content: ' + ';
}
```

**Comment by jeddy3** [comment]
Sgtm.

["Pseudo-element names are ASCII case-insensitive"](https://drafts.csswg.org/selectors-4/#pseudo-element%20syntax).

- Name: `selector-pseudo-element-case`
- Primary options: `"upper"|"lower"`
- Secondary options: none
- Message: `Expected "${actual}" to be "${expected}" (selector-pseudo-element-case)`.

We'll need to [account for](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-pseudo-element-colon-notation#selector-pseudo-element-colon-notation) single colon notation for these pseudo elements:  `:first-line`, `:first-letter`, `:before` and `:after`. They should be checked by this rule.

**Comment by evilebottnawi** [comment]
@jeddy3 WIP

**Comment by evilebottnawi** [comment]
Merged. Close.
